contrib/openshift/osba-os-template.yaml

```diff
@@ -0,0 +1,238 @@
+kind: Template
+apiVersion: v1
+objects:
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: osba-redis
+    labels:
+      app: osba-redis
+      release: "osba"
+  type: Opaque
+  data:
+    redis-password: "cGFzc3dvcmQ="
```

Comment:
This should be parameterized.

Reply:
This resource should be created conditionally, depending on whether one wants to use on-cluster Redis or a remote Redis (i.e. Azure Redis Cache).